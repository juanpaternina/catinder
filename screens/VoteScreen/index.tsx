import { View, StyleSheet } from "react-native";
import { useCatAPI } from "@/hooks/useCatAPI";
import { useEffect } from "react";
import Colors from "@/constants/Colors";

export default function VoteScreen() {
  const { getCatList } = useCatAPI();

  const fetchCats = async () => {
    try {
      const { data: cats } = await getCatList();
      console.log(JSON.stringify(cats, null, 2));
    } catch (error) {}
  };

  useEffect(() => {
    fetchCats();
  }, []);

  return <View style={styles.container}></View>;
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
    backgroundColor: Colors.default.background,
  },
  title: {
    fontSize: 20,
    fontWeight: "bold",
  },
  separator: {
    marginVertical: 30,
    height: 1,
    width: "80%",
  },
});
