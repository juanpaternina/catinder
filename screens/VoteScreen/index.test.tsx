import { render, screen } from "@testing-library/react-native";
import VoteScreen from "./index";

describe("VoteScreen", () => {
  it("renders correctly", () => {
    //render(<VoteScreen />);
  });
});
