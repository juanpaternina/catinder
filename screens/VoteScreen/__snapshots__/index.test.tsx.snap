// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VoteScreen renders correctly 1`] = `null`;
